constants.h

```diff
@@ -2,20 +2,29 @@
 #define CONSTANTS_H
 
 #include <cmath>
+#include <string>
 
 namespace constants {
 const float kAccuracy = 0.000005;
 const float kSoundSpeed = 343.216; // This is measured in 20'C at 1000 hPa
 const float kPi = std::acos(-1);
-const float kDefaultSimulationRadius = 4; // [m]
-const int kPopulation = 37;
 const float kHitAccuracy = 0.0001;
+const size_t kDefaultNumRaysPerRow = 1000;
```

Comment:
Im sure, that I won't be having problems with results of the artymetric operation being an invalid sate in this case. However maybe you are right that size_t should be only used in accesing containers such as vector, so maybe it is a good idea to change this to int, just to develop good habits.

Reply:
In my opinion, you should never use it, even with vectors and such, because this is just asking for trouble.

> "I am sure, that..."

In computer science, there is no such thing. There is a joke that people write "should not happen" comments in the code, and then they happen.